CMakeLists.txt

```diff
@@ -206,6 +228,7 @@ if(WITH_SPDK)
   set(HAVE_SPDK TRUE)
 endif(WITH_SPDK)
 
+option(WITH_BLKID "Block device library" ON)
```

Comment:
On 4-7-2016 06:18, Kefu Chai wrote:
> In CMakeLists.txt
> <https://github.com/ceph/ceph/pull/10024#discussion_r69407120>:
> 
>> @@ -206,6 +228,7 @@ if(WITH_SPDK)
>>    set(HAVE_SPDK TRUE)
>>  endif(WITH_SPDK)
>>  
>> +option(WITH_BLKID "Block device library" ON)
> 
> why would you want to make it an option?

Because it uses blkid, and that is non-existent under FreeBSD.
So for the time being I compile it, and do not run the tests.